_pages/members.md

```diff
@@ -312,6 +312,15 @@ order: 1
         <div class="caption">
             Karthik Ganesan 
         </div>	    
+        </div>
+        <div class="col-sm mt-3 mt-md-0">
+        <div class="square">
+        <a href="https://www.linkedin.com/in/debayan-ghosh-74170010a/">
+            <img class="img-fluid rounded z-depth-1" src="{{ site.baseurl }}/assets/img/debayan_ghosh.jpg">
+	</a></div>
+        <div class="caption">
+            Debayan Ghosh 
+        </div>	    
     </div>
     <div class="col-sm mt-3 mt-md-0">
```

Comment:
Remove this div would be fine (the two line 325-326)
```
<div class="col-sm mt-3 mt-md-0">
</div>
```

Reply:
updated. refer to the "remove div" commit